components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/Connection.kt

```diff
@@ -22,14 +23,14 @@ const val DB_NAME = "places.sqlite"
  *
  * Writer is always the same, as guaranteed by [PlacesApi].
  */
-interface Connection : Closeable {
+internal interface Connection : Closeable {
```

Comment:
`sync15` is an `implementation` dependency, so I don't think I can export this. Since this will likely change with the sync manager, I opted to make the interface `internal` instead of exposing a public type wrapping `SyncTelemetryPing`.